mkt/search/forms.py

```diff
@@ -177,8 +179,17 @@ def clean_device_and_dev(self):
             raise forms.ValidationError('Invalid device or device type.')
 
     def clean_author(self):
-        return self.cleaned_data.get('author', '').lower()
+        author = self.cleaned_data.get('author')
+        if author:
+            return author.lower()
 
     def clean(self):
         self.clean_device_and_dev()
+
+        # Convert empty things to `None`s.
```

Comment:
Is his always going to be right ? it seems like it is now, but we may want to differentiate between `0` and `None` in the future. Maybe not worth caring about now though.